notes/cheatsheet.md

```diff
@@ -26,3 +26,15 @@ Consume an event as conditionorc
 ```
 nats consumer sub --creds=/root/nsc/nkeys/creds/KO/controllers/conditionorc.creds -s nats://nats:4222
 ```
+
+
+list keys in statusKV buckets
```

Comment:
It's probably more useful to get the keys and examine individual data with `kv get --raw`